www/src/components/layout/page-heading.js

```diff
@@ -0,0 +1,88 @@
+import React from "react"
+import PropTypes from "prop-types"
+import styled, { css } from "react-emotion"
+
+import presets, { colors } from "../../utils/presets"
+import { rhythm, options, scale } from "../../utils/typography"
+
+const PageHeadingContainer = styled.header({
+  padding: rhythm(options.blockMarginBottom),
+
+  [presets.Tablet]: {
+    left: 0,
+    position: "fixed",
+    padding: 0,
+    top: `calc(${presets.bannerHeight} + ${presets.headerHeight})`,
+  },
+})
+
+const H1 = styled.h1({
```

Comment:
We should standardize on a way of declaring these styled components.

My preference is very strongly toward writing real CSS unless there's a good reason not to do so. I'm happy to be convinced otherwise, but I'd like to see this shifted over to the tagged template literal declaration:

```js
const Component = styled('div')`
	margin-right: 10px;
`;
```

If possible, let's avoid the JS object style declarations altogether.

Reply:
Additionally, we should make a call between `styled.div` and `styled('div')`. I don't actually care which we use as long as it's consistent. My preference is for `styled('div')` if no one else feels strongly about this.

Reply:
1. I agree, I also much more like the real CSS code with the tagged template literals, I used the objects, because nowadays styles are in objects, so after the discussion on Slack I decided to stay as close to the existing style as I can. But I gave my +1 for moving to strings.

2. https://github.com/gatsbyjs/gatsby/pull/9312#discussion_r227462113 Yes, I already did it (on my local)